### ISSUE TITLE
fix the gifs pls

### ISSUE THREAD

**Issue by raquelbreternitz** [chrisdhanaraj/radio#6]
make it so non-portrait gifs fill the entire space

**Comment by raquelbreternitz** [comment]
an idea:

should the gif crop to fit the miniplayer square

or should the miniplayer adjust to the gif size (adjust to portrait, landscape, or square) (like instagram does)

thoughts?

**Comment by chrisdhanaraj** [comment]
Ehhh I think we should crop/stretch the gif to fit the miniplayer square - otherwise you're going to have scenarios where someone uploads a gif not really knowing the dimensions and it'll look bizarre.

**Comment by raquelbreternitz** [comment]
yeah but we can still set a constraint: 600px width, variable height or 600px height, variable width, and adjust placement of the rest accordingly.

I'm cool if you want to just crop it down to a square tho, I just know sometimes that might cut off important parts of the gif.

**Comment by chrisdhanaraj** [comment]
:P fine fine - we can adjust depending on gif.
